src/routes/AccountDetailsContainer.js

```diff
@@ -65,7 +71,32 @@ class AccountDetailsContainer extends React.Component {
     instance: {
       type: 'okapi',
       path: 'instance-storage/instances/%{activeRecord.instanceId}',
-    }
+    },
+    query: { initialValue: {} },
+    feefineshistory: {
+      type: 'okapi',
+      records: 'accounts',
+      path: 'accounts',
+      recordsRequired: '%{activeRecord.records}',
+      perRequest: MAX_RECORDS,
+      GET: {
+        params: {
+          query: queryFunction(
+            'feeFineType=*',
+            'feeFineType="%{query.query}*" or barcode="%{query.query}*" or materialType="%{query.query}" or title="%{query.query}*    " or feeFineOwner="%{query.query}*" or paymentStatus.name="%{query.query}"',
```

Comment:
@Dmitriy-Litvinenko I noticed that there is an extra space around title query:

`title="%{query.query}*    "`

Is this necessary?

Reply:
Hello, I left this part unchanged when I moved it from "AccountsListingContainer.js" to "File feeFineConfig.js". Should I investigate this question as part of the tech debt?